src/wp-includes/fonts/class-wp-font-library.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Font Library class.
+ *
+ * This file contains the Font Library class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.5.0
+ */
+
+/**
+ * Font Library class.
+ *
+ * @since 6.5.0
+ */
+class WP_Font_Library {
```

Comment:
A class with only static methods doesn't make much sense from an OOP perspective and not really common in core code. What's the reason for this? Can we think of a better architecture?

Reply:
Yes, this should map the code we already have in all of our registries. Like WP_Block_Type_Registry... I'll make this update.